RELEASENOTES-1.4.md

```diff
@@ -334,3 +334,7 @@
   `#foreach` loops remains unsupported)
 - `release 7 7037`
 - `release 6 6330`
+- `note 6` Added the `destroy` template, the device deletion equivalent of the
+  `init` and `post_init` templates. When the device is being deleted,
+  `destroy()` will be called of every object that instantiates the `destroy`
+  template (the device object always instantiates it.)
```

Comment:
I'd strip the bracket to improve text flow, e.g. `destroy template. The device object always instantiates this template.`

Reply:
Makes sense.